### PR TITLE
fixed menu paths, added list

### PR DESCRIPTION
**Pull request by emilybrown1**
Part of a review of HDRP docs.

Fixed menu path formatting, added menu path to first section, made step 5 into a list.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Photoshop.md [hunk 1]
```diff
@@ -1,62 +1,67 @@
...
+2. Go to **Post-processing** > **Grading LUT Size**
+
+This is a fixed size; if you change it later, you need to rebake all your LUTs. A size of **32** provides enough precision for real-time use.
 
 ## Step 2: Render the Camera view to an EXR file
 
 Export the current [Camera](HDRP-Camera.md)'s view to a log-encoded EXR file. To do this:
 
 1. In the Scene view or Hierarchy, select a GameObject that contains a Camera component.
-2. Select **Edit** > **Rendering** > **Render Selected HDRP Camera to Log EXR** **(**or press **Ctrl+Shift+Alt+E)**.
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Photoshop.md:16]: Any reasons to remove the keyboard shortcut? this is useful for people

Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Photoshop.md:16]: hmm I notice doc is very simialr to: https://github.com/Unity-Technologies/Graphics/pull/6995/files#diff-25001ab39fff1c541102518bd4967fb93681446d0287d30e525aad8ba519d5ccL16 . Not sure why we have two page here, we should share stuff

Comment (emilybrown1) [reply com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Photoshop.md]: I've addressed these comments on the other PR

**Review by sebastienlagarde — APPROVED** [review]

**Review by Vic-Cooper — APPROVED** [review]
lgtm

**Pull request merged**